.github/workflows/tests.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       rust: ${{ !startsWith(github.ref, 'refs/pull/') || steps.filter.outputs.rust }}
+      dependencies: ${{ !startsWith(github.ref, 'refs/pull/') || steps.filter.outputs.dependencies }}
     steps:
     - uses: dorny/paths-filter@v2
```

Comment:
I think it's a little mad that this is a third-party action and not something that GHA provides by default. (Though I can see that it makes it GHA-the-system simpler and less coupled...)